pytest/tests/sanity/upgradable.py

```diff
@@ -40,6 +40,7 @@ def test_protocol_versions() -> None:
     skipped.  See <https://github.com/near/nearcore/issues/4956>.
```

Comment:
I'd also mention that this test can potentially download stuff from the internet (or build)